provider-util/src/main/java/org.wildfly.security.provider.util/ProviderFactory.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2015 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.security.provider.util;
```

Comment:
The path for this file is: ```provider-util/src/main/java/org.wildfly.security.provider.util/ProviderFactory.java```

Looks like ```org.wildfly.security.provider.util``` should be  ```org/wildfly/security/provider/util```

Reply:
@fjuma Thank you, fixed